.github/workflows/rust.yml

```diff
@@ -51,7 +51,6 @@ jobs:
                   os: "ubuntu"
                   target: "x86_64-unknown-linux-gnu"
 
-            - uses: dtolnay/rust-toolchain@stable
```

Comment:
why?

Reply:
since its already happen in Install shared software dependencies